test-cli.py

```diff
@@ -472,10 +474,10 @@ def test(ctx):
     ctx.collect_worksheet(wuuid)
     # Add tags
     run_command([cl, 'wedit', wname, '--tags', 'foo', 'bar', 'baz'])
-    check_contains(['Tags: \\[\'foo\', \'bar\', \'baz\'\\]'], run_command([cl, 'ls', '-w', wuuid]))
+    check_contains(['Tags: \\[u\'foo\', u\'bar\', u\'baz\'\\]'], run_command([cl, 'ls', '-w', wuuid]))
```

Comment:
Perhaps it's better to change the CLI code to output tags.join(' ') so that
the output looks better and we don't have to worry about this issue?

On Fri, Feb 12, 2016 at 8:05 PM, Andrew <notifications@github.com> wrote:

> In test-cli.py
> <https://github.com/codalab/codalab-cli/pull/314#discussion_r52822604>:
>
> > @@ -472,10 +474,10 @@ def test(ctx):
> >      ctx.collect_worksheet(wuuid)
> >      # Add tags
> >      run_command([cl, 'wedit', wname, '--tags', 'foo', 'bar', 'baz'])
> > -    check_contains(['Tags: \\[\'foo\', \'bar\', \'baz\'\\]'], run_command([cl, 'ls', '-w', wuuid]))
> > +    check_contains(['Tags: \\[u\'foo\', u\'bar\', u\'baz\'\\]'], run_command([cl, 'ls', '-w', wuuid]))
>
> When I run (both locally on Ubuntu 14.04 w/Python 2.7.5) and on Circle it
> says that it needs the u's. Not sure why it would output unicode strings
> now and not before
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/314/files#r52822604>.
>